server/controllers/order.js

```diff
@@ -0,0 +1,99 @@
+const express = require('express');
+const nodemailer = require('nodemailer');
+const bodyParser = require('body-parser');
+const orderModel = require('../model/order');
+const utils = require('../utils/utils');
+
+const app = express();
+
+const Logger = require('../services/logger_services');
+
+const logger = new Logger('app');
+
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+
+const transporter = nodemailer.createTransport({
+  service: 'GMAIL',
+  auth: {
+    user: 'kingmonkey409@gmail.com',
+    pass: 'Yos112233',
+  },
+});
+
+const getAllOrders = (req, res) => {
+  const ordersList = orderModel.getAllOrders();
+  logger.info('Orders were fetched', ordersList);
+  return res.status(200).json({
+    message: 'Orders were fetched',
+    ordersList,
+  });
+};
+
+const createOrder = (req, res) => {
+  const { body } = req;
+  const order = {
+    userName: body.userName,
+    product: body.product,
+    date: new Date(),
+    id: utils.idGeneretor.generateId(),
```

Comment:
The id is usually generated in the db, because you didn't have one I think this  utils.idGeneretor.generateId() need to be in the model layer. the controller didn't need to determine the id.